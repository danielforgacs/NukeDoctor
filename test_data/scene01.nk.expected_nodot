Group {
  name groupname
}
end_group
