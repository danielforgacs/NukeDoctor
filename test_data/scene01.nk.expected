Dot {
  name Dot1
}
Group {
  name groupname
}
 Dot {
  name Dot2
}
end_group
Dot {
  name Dot3
}
